packages/web/src/app.css

```
@@ -11,3 +11,59 @@
+
+  body {
+    background: #fff;
+    color: #000;
+    font-size: 11pt;
+    line-height: 1.4;
+  }
+
+  .card {
+    box-shadow: none;
+    border: 1px solid #ccc;
+    page-break-inside: avoid;
```

Stylelint (17.11.1)

[error] Expected "page-break-inside" to be "break-inside" (property-no-deprecated)

(property-no-deprecated)